data_structures.ipynb

```diff
@@ -0,0 +1,579 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Structured Data\n",
+    "\n",
+    "Data type representation is a way to identify different type of value that could be required depending on the problem to be solved. What happens when you have to repeat a certain piece of code several times? First it makes de code bigger and takes away one of the most important principles of Python coding such as readability. Also, it is heavily saying that the data _per sé_ needs to be handled in a different way. For instance:\n",
+    "\n",
+    "```python\n",
+    "\"\"\"\n",
+    "The following code has variables holding ASCII values.\n",
+    "\"\"\"\n",
+    "var_a = 97\n",
+    "var_b = 98\n",
+    "var_c = 99\n",
+    "# ...\n",
+    "var_z = 122\n",
+    "\n",
+    "# 32 is the distance from lower case chars to become upper case.\n",
+    "def to_upper(char):\n",
+    "    return char - 32\n",
+    "\n",
+    "var_A = to_upper(var_a)\n",
+    "var_B = to_upper(var_b)\n",
+    "var_C = to_upper(var_c)\n",
+    "# ...\n",
+    "var_Z = to_upper(var_z)\n",
+    "```\n",
+    "\n",
+    "There are multiple options to avoid having duplicated code and to improve how to treat your data and that will be using data structures for it to be handled accordingly. Below the data structures we will be reviewing as part of this exercise:\n",
```

Comment:
There are multiple options to avoid having duplicated code and to improve how to treat your data which can be achieved using data structures to properly handle your code. Below are the data structures we will be reviewing as part of this exercise: